cluster-http/src/test/scala/akka/cluster/http/management/scaladsl/ClusterHttpManagementRoutesSpec.scala

```diff
@@ -443,7 +449,7 @@ class ClusterHttpManagementRoutesSpec
             .futureValue(t)
         responseGetDomainEvents.status shouldEqual StatusCodes.OK
         val responseGetDomainEventsData = responseGetDomainEvents.entity.dataBytes
-          .takeWithin(100.millis)
+          .takeWithin(500.millis)
```

Comment:
I suspect this is the actual fix since taking events during a longer period is likely to obtain the one we're looking for.